src/igvfd/audit/biosample.py

```diff
@@ -0,0 +1,36 @@
+from snovault.auditor import (
+    audit_checker,
+    AuditFailure,
+)
+from .formatter import (
+    audit_link,
+    path_to_text,
+)
+
+
+@audit_checker('Biosample', frame='object')  # this audit requires metadata in other objects - donor taxa
+def audit_biosample_taxa_check(value, system):
+    '''Flag any biosample with populated donors, but without taxa meaning the donors are mixed taxas.'''
```

Comment:
if at all, I think the comment should say "Flag biosamples associated with donors of different taxas."